it/ready/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+
+go_test(
```

Comment:
missing `race="on"` and `size="small"` attributes. The first one can be added by running `make fmt-bazel` and the second one has to be added manually. Otherwise bazel prints a warning. See other `go_test` targets.